editors/vim/syntax/jakt.vim

```diff
@@ -79,6 +79,7 @@ syntax region jaktCommentLine start="//" end="$"
 
 
 syntax region jaktString matchgroup=jaktStringDelimiter start=+"+ skip=+\\\\\|\\"+ end=+"+ oneline contains=jaktEscape
+syntax region jaktChar matchgroup=jaktCharDelimiter start=+'+ end=+'+ oneline
```

Comment:
can this contain escapes also?

Reply:
If we need it, sure! I'll add another commit.